### DISCUSSION TITLE
I try to run an input file from an example it shows the error in the picture, I want to ask what is the add_material means from the error.

### DISCUSSION THREAD

**Discussion by zxj19910214** [idaholab/moose#27406] [Q&A General]
![image](https://github.com/idaholab/moose/assets/38807201/2ffa9d33-4c90-4ee0-b150-c49925c9cf27)

**Comment by GiudGiud** [comment]
Hello

You put a kernel in the `[Materials]` block

For future reference, text is best copy-pasted, not in a screenshot

Guillaume

**Reply by zxj19910214** [reply]
Dear Guillaume,

Thank you  for the answer and suggestion. I will check and modify the files.

Best regards,
Xiao

**Reply by GiudGiud** [reply]
Sounds good
Please let me know if it does not work still
